sdk-lib/src/main/java/cash/z/ecc/android/sdk/jni/RustBackend.kt

```diff
@@ -112,6 +112,16 @@ internal class RustBackend private constructor(
 
     override fun getSaplingReceiver(ua: String) = getSaplingReceiverForUnifiedAddress(ua)
 
+    override suspend fun listTransparentReceivers(account: Int): Array<String> {
```

Comment:
Should we make the account argument here be the type safe `Account` object?

It would be nice to have the type safe objects for the public API to the RustBackend (but note that we cannot use them for the `external` functions since those are for the JNI)

Reply:
Yes, I like this idea, so I went further and changed all related RustBacked methods, which used Int type to Account type. The changes were pretty straightforward, see related commit.